src/urbit/lib/chess.hoon

```diff
@@ -274,6 +274,22 @@
   --
 ++  position-to-fen
   |_  chess-position
+  ++  simplified
+    ^-  @t
+    ;:  (cury cat 3)
+      board-to-fen
+      ' '
+      ?-  player-to-move
+        %white  'w'
+        %black  'b'
+      ==
+      ' '
+      castle-to-fen
+      ' '
+      ?~  en-passant
+        '-'
+      (square-to-algebraic u.en-passant)
+    ==
```

Comment:
Reread the code example in this comment: https://github.com/ashelkovnykov/urbit-chess/pull/40#discussion_r972971479

There's a way to simplify this block and the code below it even more, so that there isn't two copies of the code which generates the first four fields of a FEN position